resources/helm/overlays/istio_cni/templates/daemonset.yaml

```
@@ -251,6 +251,64 @@ spec:
               cpu: 10m
               memory: 100Mi
 {{- end }}
+{{- end }}
+{{- if and (.Values.cni.supportedReleases) (has "v2.3" .Values.cni.supportedReleases) }}
+        - name: install-cni-v2-3
+          image: "{{ .Values.cni.image_v2_3 }}"
+          imagePullPolicy: {{ .Values.cni.imagePullPolicy }}
+          livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8000
```

Incorrect indentation.

```suggestion
          livenessProbe:
            httpGet:
              path: /healthz
              port: 8000
```

---

Thanks, I will fix it.